client/src/account/ProfileTabs.jsx

```diff
@@ -4,7 +4,6 @@ import React, {useState} from 'react';
 import TabContent from 'react-bootstrap/TabContent';
 import {Link } from 'react-router-dom';
```

Comment:
Check the spacing inside the curly braces here

Reply:
Thank you!